### ISSUE TITLE
UI hangs

### ISSUE THREAD

**Issue by oskarth** [status-im/status-console-client#58]
Often I select I contact, see some stuff in chat, type message in input and it hangs (unresponsive, no message sent, can't edit).

I also get https://github.com/status-im/status-console-client/issues/56

If there's a no ui mode where I can just type /message <topic> message and get the whole stream of replies for all messages in a log that'd be more useful for current purposes, I can just grep that in a separate terminal. Would this be doable?

https://github.com/status-im/status-console-client/issues/52

**Comment by adambabik** [comment]
Looking into it. I have the same problem.

**Comment by adambabik** [comment]
> If there's a no ui mode where I can just type /message message and get the whole stream of replies for all messages in a log that'd be more useful for current purposes, I can just grep that in a separate terminal. Would this be doable?


You can interact with the process using netcat:
```bash
$ nc -U /path/to/status-console-client/selected-dir/geth.ipc
```

And send JSON-RPC messages. I will describe it in README.

**Comment by dshulyak** [comment]
https://github.com/status-im/status-console-client/issues/50

**Comment by adambabik** [comment]
@oskarth can you test with the newest master?

**Comment by adambabik** [comment]
Fixed in #49
